.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+Please make sure that these boxes are checked before closing your pull request.
+- [ ] Set milestone for this PR
+- [ ] Add a brief explanation about this PR in CHANGELOG.md
```

Comment:
It more makes sense to me if we use past tense for the leading verb since checking box means we confirmed it's done.

Reply:
Please make sure that these boxes are checked before closing your pull request.
- [ ] Set milestone for this PR
- [ ] Added a brief explanation about this change in CHANGELOG.md
- [ ] Added unit test for this change
***
Here is an example. It is not possible to put new lines at the beginning of a template, so this layout is easier to make so far.

Reply:
I think imperatives make more sense, since we're telling people to do these things.